exp/internal/controllers/machinepool_controller_phases.go

```diff
@@ -127,12 +127,6 @@ func (r *MachinePoolReconciler) reconcileExternal(ctx context.Context, cluster *
 		return external.ReconcileOutput{}, err
 	}
 
-	// if external ref is paused, return error.
-	if annotations.IsPaused(cluster, obj) {
```

Comment:
>It doesn't make sense that the ClusterClass and MachinePool controller skip reconcileExternal if an external object is paused. So let's get rid of it

Isn't this also changing the behaviour when the cluster resource is paused, not only the external ref?